advection/conservative_sw_mhd.jl

```diff
@@ -71,7 +86,28 @@ simulation.output_writers[:fields] = JLD2OutputWriter(model, (; u, v, A = model.
                                                       overwrite_existing = true)
 
 @info "Running with Δt = $(prettytime(simulation.Δt))"
+sim_start_time = time_ns()*1e-9
 run!(simulation)
+sim_end_time = time_ns()*1e-9
+sim_time = prettytime(sim_end_time - sim_start_time)
+@info "Simulation took $(sim_time) to finish running."
+
+uh, vh, h = simulation.model.solution
```

Comment:
Do yoiu need to do all of this again?

Reply:
You need to define these fields to be able to write to the file.  You seem to be computing it on the initial conditions and the final conditions, but do you use both of those?  If you define it for the output writers, then you can read it in the data at all time steps, and that should suffice.